### PR TITLE
Ses dev

### PR DESCRIPTION
**Pull request by benwilson512**

### CODE REVIEW

File: lib/ex_aws/ses.ex [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  end
+
+
+  ## Emails
+  ######################
+  @type message :: %{
+    body: %{html: %{data: binary, charset: binary}, text: %{data: binary, charset: binary}},
+    subject: %{data: binary, charset: binary}
+  }
+  @type destination :: %{to: [binary], bcc: [binary], bcc: [binary]}
```
Comment (benwilson512) [inline lib/ex_aws/ses.ex]: @lessless Do note that I shortened the API here a bit from `to_addresses` and so on to just `to`. They're all addresses, I didn't feel like suffixing them added value.

File: lib/ex_aws/ses.ex [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    request(:get_identity_verification_attributes, params)
+  end
+
+  @type list_configuration_sets_opt :: {:max_items, pos_integer}
+    | {:next_token, String.t}
+
+  @doc "Fetch configuration sets associated with AWS account"
+  @spec list_configuration_sets() :: ExAws.Operation.Query.t
+  @spec list_configuration_sets(opts :: [] | [list_configuration_sets_opt]) :: ExAws.Operation.Query.t
+  def list_configuration_sets(opts \\ []) do
```
Comment (benwilson512) [inline lib/ex_aws/ses.ex]: @lessless note that this went from `configuration_sets` to `list_configuration_sets` to more closely match the AWS naming.

File: lib/ex_aws/ses.ex [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+  @doc "Verifies an email address"
+  @spec verify_email_identity(email :: binary) :: ExAws.Operation.Query.t
+  def verify_email_identity(email) do
+    request(:verify_email_identity, %{"EmailAddress" => email})
+  end
+
+  @doc "Fetch identities verification status and token (for domains)"
+  @spec get_identity_verification_attributes([binary]) :: ExAws.Operation.Query.t
+  def get_identity_verification_attributes(identities) when is_list(identities) do
```
Comment (benwilson512) [inline lib/ex_aws/ses.ex]: @lessless note that this went from `identity_verification_attributes` to `get_identity_verification_attributes` to more closely match AWS naming

**Comment by lessless** [comment]
@benwilson512 thanks!

File: lib/ex_aws/ses.ex [hunk 4]
```diff
@@ -72,9 +73,9 @@ defmodule ExAws.SES do
 
   defp format_tags(tags) do
     tags
-    |> Enum.with_index
+    |> Enum.with_index(1)
```
Comment (lessless) [inline lib/ex_aws/ses.ex]: Ha, that's something new today 👍

**Pull request merged**